src/config_parse/config.cpp

```diff
@@ -0,0 +1,99 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   config.cpp                                         :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: lizhang <lizhang@student.codam.nl>           +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2023/06/26 16:49:22 by lizhang       #+#    #+#                 */
+/*   Updated: 2023/06/29 16:17:02 by lizhang       ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../../include/config.hpp"
+#include <fstream>
+#include <iostream>
+
+static std::string getcontent(std::string file_content, std::string title)
+{
+	size_t	pos_start;
+	size_t	pos_end;
+
+	pos_start = file_content.find(title, 0);
+	pos_start = pos_start + title.length();
+	pos_start = file_content.find("{", pos_start);
+	pos_start ++;
+	pos_end = file_content.find("}", pos_end);
+	return(file_content.substr(pos_start, pos_end));
+}
+
+static std::string get_value(std::string content, std::string title)
+{
+	std::string line;
+	size_t		pos = 0;
+	size_t		pos2;
+
+	while (pos < content.length())
+	{
+		pos2 = content.find(";", pos);
+		line = content.substr(pos, pos2 - pos);
+		if (line.find(title) < line.length())
+			break ;
+		pos = pos2 + 1;
+	}
+	pos = line.find(" ");
+	return (line.substr(pos + 1, line.length() - pos - 1));
+}
+
```

Comment:
A very smart way to get the directive(content) and value!
I will add another config files later, there will be more valuables we have to store, and some valuables would be multiple elements. In this moment, just modifying the names of class, member functions and valuables is enough I think...